webapp/src/shared/components/Score/Score.tsx

```diff
@@ -2,13 +2,13 @@ import { AvatarProps, MediumAvatar } from '../Avatar/Avatar';
 import Text, { TextColor, TextVariant, TextWeight } from '../Text/Text';
 import './Score.css';
 
-type ScoreProps = {
-  playerOneAvatar: AvatarProps;
-  playerTwoAvatar: AvatarProps;
+export type ScoreProps = {
+  playerOneAvatar?: AvatarProps;
+  playerTwoAvatar?: AvatarProps;
   playerOneUserName: string;
   playerTwoUserName: string;
-  playerOneScore: number;
-  playerTwoScore: number;
+  playerOneScore?: number;
+  playerTwoScore?: number;
```

Comment:
question: This might be a left-over, or do we want to allow passing `undefined` scores to this component?

Reply:
It is not a left-over, I was thinking in using this component for the ongoing games, and without the scores